Program.cs

```diff
@@ -3,17 +3,57 @@
 // Copyright (c) Metamation India.                                                
 // ---------------------------------------------------------------------------
 // Program.cs                                                                     
-// 
-// 
-// 
+// Program to swap the indices of a given list of numbers
+// A random series of numbers is displayed to the user and the user is asked to provide the two indices to be swapped.
+// The number series after swapping the indices is displayed.
 // ---------------------------------------------------------------------------------------
-using static System.Console;
+
 namespace Training {
    #region class Program ------------------------------------------------------------------------
+   /// <summary>Swap Indices</summary>
    internal class Program {
       #region Method ----------------------------------------------
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+      /// <summary>This method prints a series of numbers based on the count given by the user and displays the updated series after swapping the user asked indices</summary>
+      static void Main () {
+         Console.WriteLine ("Enter the count (2-10) of random numbers to be displayed: ");
+         int count = ValidInputCheck ();
+         if (count < 2) Console.WriteLine ("You atleast need two numbers to swap indices.");
+         else {
+            int[] numberSeries = new int[count];
+            for (int i = 0; i < count; i++) {
+               Random random = new ();
+               numberSeries[i] = random.Next (1, 10);
+            }
+            foreach (int number in numberSeries) Console.Write (number + " ");
+
+            Console.WriteLine ("\nEnter the indices to be swapped: \nFirst Index: ");
+            try {
+               int firstIndex = ValidInputCheck ();
+               Console.WriteLine ("Second Index: ");
+               int secondIndex = ValidInputCheck ();
+               SwapIndices (numberSeries, firstIndex, secondIndex);
+               foreach (int number in numberSeries) Console.Write (number + " ");
+            } catch { Console.WriteLine ("One or two of the given indices were out of range."); }

```

Comment:
The right way to use catch would be `catch (Exception e) { Console.WriteLine (e.Message); }` as the exception type can vary. If you want to display a message that relates to index out of range, then it should be `catch (IndexOutOfRangeException) { Console.WriteLine ("One or two of the given indices were out of range."); }`. Note that usage of multiple catch blocks is possible.